src/charm.py

```diff
@@ -16,11 +17,18 @@
 logger = logging.getLogger(__name__)
 
 WEB_UI_PORT = 8080
+INTERNAL_API_PORT = 8001
+CONNECTOR_BUILDER_API_PORT = 80
 
 
 class AirbyteUIK8sOperatorCharm(CharmBase):
     """Charm the application."""
 
+    @property
+    def external_hostname(self):
+        """Return the DNS listing used for external connections."""
+        return self.config["external-hostname"] or self.app.name
```

Comment:
question (non-blocking): Does `self.config["external-hostname"]` ever return a falsy value?

It has a default in the charm config, do you have to explicitly set it to an empty string for it to be falsy?

Reply:
Yes, if the config is explicitly set to`""` then it will return a falsy value.